android/settings.gradle

```diff
@@ -8,6 +8,10 @@ include ':reactnativenotifications'
 project(':reactnativenotifications').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-notifications/lib/android/app')
 include ':watermelondb-jsi'
 project(':watermelondb-jsi').projectDir = new File(rootProject.projectDir, '../node_modules/@nozbe/watermelondb/native/android-jsi')
+include ':frameanimation'
+project(':frameanimation').projectDir = new File('../node_modules/APNG4Android/frameanimation')
+include ':gif'
+project(':gif').projectDir = new File('../node_modules/APNG4Android/gif')
```

Comment:
In order to get this as project in expo-image/android/build.gradle, need to declare `frameanimation` and `gif` here.